src/shared/components/common/markdown-textarea.tsx

```diff
@@ -328,6 +331,21 @@ export class MarkdownTextArea extends Component<
     );
   }
 
+  handleEmoji(i: MarkdownTextArea, e: any) {
+    let value = e.native;
+    if (value == null){
+      let lookup = getCustomEmojiLookup()
```

Comment:
If you've already built the map, just export that value and refer to it, no need to call functions that might accidentally re-initialize things.